### PR TITLE
Update triton-vm to use latest twenty-first

### PR DESCRIPTION
**Pull request by sshine**
Update triton-vm to use latest twenty-first
    
- Neptune-Crypto/twenty-first#36: ::variables() had its second argument removed.
- Neptune-Crypto/twenty-first#42: GetRandomElements has been removed.

### CODE REVIEW

File: triton-vm/Cargo.toml [hunk 1]
```diff
@@ -26,7 +26,7 @@ version = "1"
 default-features = false
 
 [dependencies]
-twenty-first = { version = "0.2.0" } # , path = "../twenty-first" }
+twenty-first = { git = "https://github.com/Neptune-Crypto/twenty-first.git", branch = "random-elements" }
```
Comment (sshine) [inline triton-vm/Cargo.toml:29]: Before merging the present PR, merge this branch on twenty-first and change the dependency to the master branch.

Comment (jan-ferdinand) [reply triton-vm/Cargo.toml:29]: Why not make a new release on crates.io and depend on that? My impression was that depending on git repositories was unusual.

**Comment by jan-ferdinand** [comment]
An additional feature present in the latest version of twenty-first is, afaik, `inverse_or_zero()` of field elements. It might be nice to start calling those methods where it makes sense. However, I'm not sure this PR is the right place for that change.

File: triton-vm/Cargo.toml (hunk 1 above)
Comment (sshine) [reply triton-vm/Cargo.toml:29]: I'd like to do that once the digest type has been added to twenty-first.

**Comment by sshine** [comment]
> It might be nice to start calling those methods where it makes sense.

That would be a nice addition. This PR isn't doing much, I've added it as a separate commit.

**Pull request merged**